packages/sns/lib/sns/SnsPublisherManager.spec.ts

```diff
@@ -50,7 +43,13 @@ describe('SnsPublisherManager', () => {
       await fakeConsumer.start()
 
       // When
-      const publishedMessage = await publisherManager.publish(TestEvents.created.snsTopic, message)
+      const publishedMessage = await publisherManager.publish(TestEvents.created.snsTopic, {
+        ...publisherManager.resolveBaseFields(),
```

Comment:
I wish we will be able to figure out a better way, that would both preserve specific payload type here, and not require explicitly passing all of the core fields, but so far I wasn't very successful in this.